docs/content/docs/tutorials/ec2-example.md

```diff
@@ -0,0 +1,477 @@
+kubectl describe subnets
+kubectl describe securitygroups
+```
+ 
+* Subnet gets into an 'available' state with a `ACK.ReferencesResolved = True` condition attached notifying users that the references (VPC, RouteTable) have been found and resolved:
+ 
+```
+Status:
+  Ack Resource Metadata:
+    Arn:                       arn:aws:ec2:us-west-2:<ID>:subnet/subnet-0ba22f5820bb41584
+    Owner Account ID:          <ID>
+    Region:                    us-west-2
+  Available IP Address Count:  4091
+  Conditions:
+    Last Transition Time:           2022-10-13T14:54:39Z
+    Status:                         True
+    Type:                           ACK.ReferencesResolved
+    Last Transition Time:           2022-10-13T14:54:41Z
+    Message:                        Resource synced successfully
+    Reason:
+    Status:                         True
+    Type:                           ACK.ResourceSynced
+  Default For AZ:                   false
+  Map Customer Owned IP On Launch:  false
+  Owner ID:                         515336597380
+  Private DNS Name Options On Launch:
+  State:      available
+  Subnet ID:  subnet-<ID>
+ 
+```
+ 
+### Validate
+ 
+This network setup should allow Instances deployed in the Private Subnet to connect to the internet. To validate this behavior deploy an Instance into the Private Subnet and the Public Subnet (bastion host). After deployments, `ssh` into the bastion host, then `ssh` into the Private Subnet Instance, and test internet connection. Security group is required by both instances launched in public and private subnets. 
+
+Note, we need to provide Subnet and SecurityGroup ID's in the yaml manually; run `kubectl describe subnets` and `kubectl describe securitygroups` commands to get ID's.
+ 
+* Deploy an Instance into the Private Subnet using provided yaml and `kubectl apply -f instance-private-subnet.yaml`:
+ 
+```
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: Instance
+metadata:
+  name: tutorial-instance-private
+spec:
+  imageID: ami-02b92c281a4d3dc79 # AL2; us-west-2
+  instanceType: c3.large
+  subnetID: subnet-<private-ID>
+  securityGroupIDs:
+  - sg-<ID>
+  keyName: us-west-2-key # created via console
+  tags:
+    - key: producer
+      value: ack
+```
+ 
+* Deploy an Instance into the Public Subnet using provided yaml and `kubectl apply -f instance-public-subnet`:
+ 
+```
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: Instance
+metadata:
+  name: tutorial-instance-public
+spec:
+  imageID: ami-02b92c281a4d3dc79 # AL2 in us-west-2
+  instanceType: c3.large
+  subnetID: subnet-<public-ID>
+  securityGroupIDs:
+  - sg-<ID>
+  keyName: us-west-2-key # created via console
+  tags:
+    - key: producer
+      value: ack
+```
+ 
+* Deployed 2 instances; one to each Subnet
+  * The instance in the public subnet will be the bastion host so we can ssh to the Instance in the private Subnet
+    ```bash
+    scp "/path/created_key_in_console_for_region.pem" ec2-user@<Public IPV4 DNS>:
```

Comment:
Note that `scp` also needs to supply the key for authentication, in addition to trying to copy the key file. The `ssh` commands on the following lines already have the necessary `-i` parameter; we should add it here too.

The command should look like this (it's a little unwieldy, unfortunately...):

```suggestion
    scp -i "/path/created_key_in_console_for_region.pem" "/path/created_key_in_console_for_region.pem" ec2-user@<Public IPV4 DNS>:
```